packages/tezos-client/tezos-client.8.1/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: [ "Tezos devteam" ]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "dune" { >= "2.0" }
+
+  "tezos-client-base-unix" { = version }
+  "tezos-node" { with-test & = version }
+  "tezos-protocol-compiler" { with-test & = version }
+  "tezos-protocol-alpha-parameters" { with-test & = version }
+
+  "tezos-client-genesis-carthagenet" { = version }
+  "tezos-client-000-Ps9mPmXa" { = version }
+  "tezos-client-007-PsDELPH1-commands-registration" { = version }
+  "tezos-client-008-PtEdoTez-commands-registration" { = version }
+  "tezos-baking-007-PsDELPH1-commands" { = version }
+  "tezos-baking-008-PtEdoTez-commands" { = version }
+]
+depopts: [
+  "tezos-client-genesis" { = version }
+  "tezos-client-demo-counter" { = version }
+  "tezos-client-alpha-commands-registration" { = version }
+  "tezos-baking-alpha-commands" { = version }
+
+  "tezos-client-001-PtCJ7pwo-commands" { = version }
+  "tezos-client-002-PsYLVpVv-commands" { = version }
+  "tezos-client-003-PsddFKi3-commands" { = version }
+  "tezos-client-004-Pt24m4xi-commands" { = version }
+  "tezos-client-005-PsBabyM1-commands" { = version }
+  "tezos-client-006-PsCARTHA-commands" { = version }
+]
```

Comment:
Constraints in the `depopts` field have a different behaviour compared to the `conflicts` field.
The former allows the depopts to be installed alongside this package if they don't have the same version. This is obviously wrong in most cases.

In this particular case it might be held up together by one thread as everything is constrained down to some tezos base package

Could you use the `conflicts` field instead for this package and others like it which do the same thing?

Reply:
Done, Thank you